src/Facebook/FacebookRedirectLoginHelper.php

```diff
@@ -100,6 +100,10 @@ public function getLoginUrl($scope = array(), $version = null, $displayAsPopup =
       'sdk' => 'php-sdk-' . FacebookRequest::VERSION,
       'scope' => implode(',', $scope)
     );
+
+    if (!empty($reauthenticate)) {
```

Comment:
Why the empty() check rather than just truthiness?  And once inside the if, why the ternary condition instead of just setting it.